src/Platform.Handlers/src/Xamarin.Platform.Handlers/Platform/Android/LayoutViewGroup.Android.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using Android.Content;
+using Android.Runtime;
+using Android.Util;
+using Android.Views;
+using Rectangle = Xamarin.Forms.Rectangle;
+using Size = Xamarin.Forms.Size;
+
+namespace Xamarin.Platform.Handlers
+{
+	public class LayoutViewGroup : ViewGroup
+	{
+		public LayoutViewGroup(Context context) : base(context)
+		{
+		}
+
+		public LayoutViewGroup(IntPtr javaReference, JniHandleOwnership transfer) : base(javaReference, transfer)
```

Comment:
Are these here intentionally for cases where the layout will be inflated from an XML file?